setup.cfg

```diff
@@ -30,6 +30,8 @@ install_requires =
     einops
     # fiona 1.5+ required for fiona.transform module
     fiona>=1.5
+    # kornia latest required for mypy fixes
```

Comment:
We only ever use the latest version when testing mypy anyway, mypy issues aren't reasons to make a version required, I would go back to the previous requirements in setup.cfg and environment.yml but install a newer version just for CI.

Reply:
Reverted and added to the tests github action under the mypy run section.